### PR TITLE
Resolve Mainframer version during build.

### PR DESCRIPTION
**Pull request by artem-zinnatullin**
Part of #192.

Docs: https://doc.rust-lang.org/cargo/reference/environment-variables.html

Thanks @pepyakin for the link!

### CODE REVIEW

**Review by vanniktech — APPROVED** [review]
Nice

**Comment by artem-zinnatullin** [comment]
@vanniktech appreciate your interest and reviews 😽

File: src/main.rs [hunk 1]
```diff
@@ -14,7 +14,7 @@ use std::time::Instant;
 use time::*;
 
 fn main() {
-    println!(":: Mainframer v2.1.0\n");
+    println!(":: Mainframer v{}\n", env!("CARGO_PKG_VERSION"));
```
Comment (arturdryomov) [inline src/main.rs:17]: How does it even work though? When runtime is packaged as a binary it exports environment variables itself?

Comment (artem-zinnatullin) [reply src/main.rs:17]: `env!` is a macro that compiler executes at build time, `CARGO_PKG_VERSION` is an environment variable provided by `cargo` to `rustc` during the build, so as a result we get version inlined into the code and binary

Comment (arturdryomov) [reply src/main.rs:17]: These native folks will steal your soul eventually, you know. Avoid blood-involved rituals at all costs.

**Review by arturdryomov — APPROVED** [review]

File: src/main.rs (hunk 1 above)
Comment (artem-zinnatullin) [reply src/main.rs:17]: lol, will try

btw Kotlin might get macros support eventually

**Pull request merged**